src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationAPI.js

```diff
@@ -34,26 +34,30 @@ type UseAnnotationAPI = {
     handleDelete: ({ id: string, permissions: AnnotationPermission }) => any,
     handleEdit: (id: string, text: string, permissions: AnnotationPermission) => void,
     handleStatusChange: (id: string, status: FeedItemStatus, permissions: AnnotationPermission) => void,
+    initialReplies: Array<Comment>,
```

Comment:
We don't seem to be using this prop anywhere, though we have state with the same name.